_topic_map.yml

```diff
@@ -240,6 +240,8 @@ Topics:
 - Name: Installing on vSphere
   Dir: installing_vsphere
   Topics:
+  - Name: Installing a cluster on vSphere
```

Comment:
This is repeated. i.e. there are two sections named `Installing a Cluster on vSphere`